lib/rouge/lexers/python.rb

```diff
@@ -8,7 +8,7 @@ class Python < RegexLexer
       desc "The Python programming language (python.org)"
       tag 'python'
       aliases 'py'
-      filenames '*.py', '*.pyw', '*.sc', 'SConstruct', 'SConscript', '*.tac'
+      filenames '*.py', '*.pyw', '*.sc', 'SConstruct', 'SConscript', '*.tac', '*.bzl', 'BUCK', 'BUILD', 'BUILD.bazel', 'WORKSPACE'
```

Comment:
`BUILD` and `WORKSPACE` seem like very generic phrases that other users might use without any intention of them being lexed as Python (arguably the case with `BUCK` as well).

Could these all be dropped and just `*.bzl` and `Build.bazel` be added instead?

Reply:
`BUILD` and `WORKSPACE` are some of the primary files used by Bazel.

A lot of the open source rules, e.g. [rules_docker](https://github.com/bazelbuild/rules_docker), especially those made my Google folks, tend to use `BUILD` instead of `BUILD.bazel.`

It seems like fortunately [Bazel also supports `WORKSPACE.bazel`](https://docs.bazel.build/versions/master/build-ref.html#workspace), which is great for our repos, but I don't think I'd be able to convince all the other rules repositories to rename all their files.

I guess GitHub [properly highlighting](https://github.com/bazelbuild/rules_docker/blob/master/WORKSPACE) WORKSPACE files won't change your mind.

I guess either those people using BUILD and WORKSPACE that don't want them highlighted or us that want them highlighting could do some `.gitattributes` hackery on GitLab to get them properly highlighted at least, or we can just maintain a patched Rouge on our forked GitLab, though I'd prefer not having to do any of those.

From the Googling I've done, it doesn't seem like any open source tools use BUILD or WORKSPACE files

Reply:
The concern was that other users would have files they expected to be lexed as plaintext suddenly show up as lexed with Python. I'm unfamiliar with Bazel and if I had a file on GitLab called `WORKSPACE` and it had syntax highlighting in it, I'd be confused.

That said, the fact that GitHub lexes these files as Python is persuasive. While not determinative in and of itself, I think it's generally a good idea for Rouge's approach to be consistent with other lexers (such as GitHub's).

I've slightly reformatted the line and added some assertions to the Python spec. You didn't have anything else you needed to add, did you?

Reply:
I wanted to add some tests, but I think the ones you added are great.

Reply:
Great!